### PR TITLE
Dust off project

### PR DESCRIPTION
**Pull request by kylerisse**
Cleaned up project. Now it works with modern go. Should be enough to get local dev up for the following 2 endpoints:

- http://localhost:8080/schedule/
- http://localhost:8080/img/

Lots to do. Would love to see a basic front end that can display the schedule and start to collaborating on how to move forward together. @kurikurichan @sarcasticadmin @carlynlee Feel free to file issues or schedule some time to talk.

### CODE REVIEW

File: signs/config.go [hunk 1]
```diff
@@ -1,111 +1,19 @@
...
-)
-
 // Config server configuration
 type Config struct {
 	Address            string
 	ScheduleXMLurl     string
 	ScheduleXMLupdate  string
 	ScheduleJSONurl    string
 	ScheduleJSONupdate string
-	TwitterJSONurl     string
```
Comment (sarcasticadmin) [inline signs/config.go:18]: Assuming the twitter is TBD given the changes to twitter's api within the last year?

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,8 @@
+module github.com/kylerisse/go-signs
+
+go 1.21.6
```
Comment (sarcasticadmin) [inline go.mod:3]: fresh version of go!

File: README.md [hunk 3]
```diff
@@ -1,3 +1,13 @@
 # go-signs
 
-POC Signs application for SCaLE. Not in a useable state. Learned some interesting things along the way, some worth repeating, some not worth repeating. No plans to continue work on this but there might be intresting portions worth looking at again.
+`go-signs` is a service that provides an API layer for the Southern California Linux Expo signage. The long term intention is to provide all schedule, twitter, sponsors, and images for consumption by a dedicated front end. (and hopefully new features down the road!) This will allow for quicker iteration than the current solution in [scale-signs](https://github.com/socallinuxexpo/scale-signs) since data and formatting will be located at separate tiers.
```
Comment (sarcasticadmin) [inline README.md:3]: thanks for updating the description

File: Makefile [hunk 4]
```diff
@@ -71,3 +49,8 @@ clean:
 mrproper: clean
 	# make: mrproper
 	rm -f $(BINARY)-v*.tar.gz
+
+.PHONY: deps
+deps:
```
Comment (sarcasticadmin) [inline Makefile:54]: i can never remember these, thanks for the recipe

**Review by sarcasticadmin — COMMENTED** [review]
I know its already merged, but thanks for getting this updated. Im really looking forward to what well have by 22x!

File: signs/config.go (hunk 1 above)
Comment (kylerisse) [reply signs/config.go:18]: Exactly. The dependencies were deprecated and it was a bit bloated. Will need to tackle the Twitter API changes but really want to understand the goals before we do that. Right now just laser focus on schedule and then sponsors.

**Pull request merged**